core/gui/src/app/hub/component/workflow/search/hub-workflow-search.component.scss

```diff
@@ -1,20 +1,34 @@
-.container {
-  background-color: #f4f4f4;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+$section-height: 76px;
+$section-margin: 24px;
```

Comment:
Try to avoid references in CSS; it's possible to set margin or padding from parents instead of aligning those child complements manually.